pallets/xvm/src/benchmarking.rs

```diff
@@ -55,10 +61,12 @@
         let source = whitelisted_caller();
         let target = MultiAddress::<T::AccountId, ()>::Id(whitelisted_caller()).encode();
         let input = vec![1, 2, 3];
+        let value = 1_000_000u128;
 
         #[block]
         {
-            Pallet::<T>::call_without_execution(context, vm_id, source, target, input).unwrap();
+            Pallet::<T>::call_without_execution(context, vm_id, source, target, input, value)
+                .unwrap();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous review comment, the code is missing validation for the length of the `target` parameter. Please add the same validation check as mentioned earlier to ensure the length of the `target` is valid.


<!-- This is an auto-generated comment by OSS CodeRabbit -->